python/cuml/dask/common/base.py

```diff
@@ -222,6 +224,54 @@ def _inverse_transform(self, X, n_dims=1, delayed=True, **kwargs):
                                        **kwargs)
 
 
+class SyncFitMixinLinearModel(object):
+
+    def _fit(self, model_func, data, **kwargs):
```

Comment:
we use it in line 247, and it is so that we do not need to change the function API signature when a model has more or less arguments needed for model building

Reply:
Actually for some reason I thought we were using **kwargs and not self.kwargs there, so you're right, will remove this